cpmonitor/templates/city.html

```diff
@@ -78,18 +78,8 @@ <h3>KPIs</h3>
           {% if charts %}
             <div class="card">
               <div class="card-body">
-                <div id="charts" class="carousel slide carousel-fade carousel-dark">
+                <div id="charts" class="carousel slide carousel-dark">
                   {% with first_chart=charts.first %}
-                    <div class="carousel-indicators-thumb">
```

Comment:
I kinda liked the thumbnails...

Reply:
Sorry. ;-)
Personally I don't like them at all as they clutter the page and don't add extra value (too small to recognise anything), but that was not the reason to remove them here. I am aware of our differing opinion on the carousel and that it will be on the agenda for the next meeting. Postpone the merge if you prefer until after the meeting.

Reply:
What was the reason to remove them?

Reply:
They use inline styles to set the thumbnails. So it was quickest to just remove them. If it is decided to put them back in, we can find a solution for the inline styles then.

Reply:
Ah! That is why you did it in this PR! Thx for your patience with me.